gradle/run.gradle

```diff
@@ -45,6 +45,17 @@ testClusters {
         plugin('plugins:'.concat(p))
       }
     }
+    setting 'opensearch.experimental.feature.read.write.split.enabled', 'true'
```

Comment:
I will remove these settings once I have the PR ready for review, these changes are to ensure I can test scale to zero implementation with remote store and read write split enabled.